modules/admin/proc-running-rhdh-behind-a-proxy.adoc

```diff
@@ -0,0 +1,132 @@
+[id='proc-running-rhdh-behind-a-proxy_{context}']
+= Running the {product-very-short} application behind a corporate proxy
+
+You can run the {product-very-short} application behind a corporate proxy by setting any of the following environment variables before starting the application:
+
+* `HTTP_PROXY`: Denotes the proxy to use for HTTP requests.
+* `HTTPS_PROXY`: Denotes the proxy to use for HTTPS requests.
+
+Additionally, you can set the `NO_PROXY` environment variable to exclude certain domains from proxying. The variable value is a comma-separated list of hostnames that do not require a proxy to get reached, even if one is specified.
+
+For Helm-based deployment, both a developer or a cluster administrator, with permissions to create resources in the cluster, can configure the proxy variables in a `values.yaml` Helm configuration file.
+
+For Operator-based deployment, the approach you use for proxy configuration is based on your role:
+
+* As a cluster administrator with access to the Operator namespace, you can configure the proxy variables in the Operator's default ConfigMap file. This configuration applies the proxy settings to all the users of the Operator.
+* As a developer, you can configure the proxy variables in a custom resource (CR) file. This configuration applies the proxy settings to the {product-very-short} application created from that CR.
+
+
+.Prerequisites
+
+* You have installed the {product} application. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
```

Comment:
```suggestion
* You have installed the {product} application. For more information, see xref:proc-install-rhdh_rhdh-getting-started[Administration guide for {product}].
```

Otherwise it shows up as _see [proc-install-rhdh_rhdh-getting-started]_ in the preview. 
Although I think you could also remove this xref entirely. It isn't really clear what the user is supposed to look at or gain from this guide.